packages/Frontend/src/components/__snapshots__/Authenticate.test.jsx.snap

```diff
@@ -205,12 +237,28 @@ exports[`Authenticate should render an error when passed as a prop 1`] = `
           Sign In
         </Button>
       </Link>
+      <Button
+        as="button"
+        color="blue"
+        fluid={true}
+        onClick={[Function]}
+      >
+        Sign In With Google
+      </Button>
+      <Button
+        as="button"
+        color="blue"
+        fluid={true}
+        onClick={[Function]}
+      >
+        Sign In With Facebook
+      </Button>
     </Form>
     <Message
       negative={true}
     >
       <p>
-        test error
+        Could not sign in, please check your email and password and try again
```

Comment:
This error came from the switch case function I made to handle errors. If it doesn't receive a valid error code for some reason this is the default message, because in my test I didn't give it a valid error code.